### PR TITLE
Completion functions for bash and zsh

### PR DESCRIPTION
**Pull request by rbvigilante**

### CODE REVIEW

File: static/completions/_yak.zsh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#compdef _yak yak
+
+function _yak {
+    local roles=($(yak --list-roles --cache-only 2>/dev/null))
+    _arguments '-h[Display this help message and exit]' \
+               '--help[Display this help message and exit]' \
+               '-l[List all available AWS roles and exit]' \
+               '--list-roles[List all available AWS roles and exit]' \
+               '-u[Your Okta Username]:username' \
+               '--okta-username[Your Okta Username]:username' \
```
Comment (toothbrush) [inline static/completions/_yak.zsh]: Shouldn't `Username` be lowercased here?

Comment (rbvigilante) [reply static/completions/_yak.zsh]: Probably. I copied it from the helptext in the application; I'll fix it there too.

**Comment by toothbrush** [comment]
The Bash is incomprehensible, but it all LGTM! 🇨🇻

**Comment by rb-johnmurphy** [comment]
LGTM!

**Pull request merged**